inorbit_edge/robot.py

```diff
@@ -960,7 +960,7 @@ def connect(self):
         # TODO: add support for user-provided CA certificate file.
         if self.use_ssl:
             self.logger.debug("Configuring client to use SSL")
-            self.client.tls_set(certifi.where(), tls_version=ssl.PROTOCOL_TLSv1_2)
+            self.client.tls_set(certifi.where(), tls_version=ssl.PROTOCOL_TLS_CLIENT)
```

Comment:
nit: please add the same comment you added in the PR description about what this constant does